pkgs/tools/misc/nncp/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, go, fetchurl }:
+
+stdenv.mkDerivation rec {
```

Comment:
Why don't you use `buildGoModule` or `buildGoPackage`?

Reply:
I'm not familiar with the Go ecosystem and just copied what I've found at https://github.com/jgoerzen/nncp

Reply:
Thanks for linking the repo. I guess in this case where they just bundle all the necessary code in `src/`, and they version that code themselves, it's Ok to use the standard environment's builder.